src/webportal/src/app/job/job-view/fabric/JobList/index.jsx

```diff
@@ -70,6 +70,23 @@ export default function JobList() {
 
   useEffect(() => filter.save(), [filter]);
 
+  useEffect(() => {
+    const query = querystring.parse(location.search.replace(/^\?/, ''));
+    if (query['vcName']) {
+      fetch(`${webportalConfig.restServerUri}/api/v1/virtual-clusters`)
```

Comment:
There are 2 virtual-cluster requests, could you please merge into only 1? Maybe move this effect to above Topbar effect